.github/workflows/test.yml

```diff
@@ -31,16 +31,23 @@ jobs:
 
       - name: Run Unit Tests
         run: |
-          pnpm run ci --coverage
+          pnpm test:coverage
```

Comment:
https://github.com/mermaid-js/mermaid/pull/4500#discussion_r1233031021

Reply:
I think it'll automatically detect CI and run it properly.